pcdsdevices/epics_motor.py

```diff
@@ -52,20 +51,18 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
         4. The description field keeps track of the motors scientific use along
            the beamline.
     """
-
     # Enable/Disable puts
     disabled = Cpt(EpicsSignal, ".DISP", kind='omitted')
     # Description is valuable
     description = Cpt(EpicsSignal, '.DESC', kind='normal')
     # Current Dial position
     dial_position = Cpt(EpicsSignalRO, '.DRBV', kind='normal')
 
-    tab_whitelist = ["set_current_position", "home", "velocity",
-                     "enable", "disable", "check_limit_switches"]
+    tab_whitelist = ["set_current_position", "home", "velocity", "enable",
+                     "disable", "check_limit_switches", "get_low_limit",
+                     "set_low_limit", "get_high_limit", "set_high_limit"]
 
     def __init__(self, *args, **kwargs):
```

Comment:
Looks like `__init__` can be removed here?

Reply:
Did we only have this because of the `_limits`?

Reply:
took it out in 54b6349